src/taipy/core/taipy.py

```diff
@@ -497,3 +498,51 @@ def export_scenario(
         _ScenarioManagerFactory._build_manager()._export(scenario_id, folder_path)
     for job_id in entity_ids.job_ids:
         _JobManagerFactory._build_manager()._export(job_id, folder_path)
+
+
+def get_parents(
+    entity: Union[TaskId, DataNodeId, PipelineId, Task, DataNode, Pipeline], parent_dict=None
+) -> Dict[str, Set[_Entity]]:
+    """Get parents of the entity from the entity or its identifier.
+
+    Parameters:
+        entity (Union[TaskId^, DataNodeId^, PipelineId^, Task, DataNode, Pipeline]): The entity of its identifier
+            to get the parent.<br/>
+    Returns:
+        Dict[str, Set[_Entity]]: The dictionary of parents
+        matching the corresponding entity. An empty dictionary if the entity does not have parents.
```

Comment:
We should describe the structure of the dictionary. What are the keys?

Reply:
I have changed it to `The dictionary of parents matching the corresponding entity with the key is the level of the parent for example 'scenarios', 'pipelines', 'tasks' and the value is a list of the parent entities.`. What do you think?